jmx-metrics/src/integrationTest/java/io/opentelemetry/contrib/jmxmetrics/target_systems/JvmTargetSystemIntegrationTest.java

```diff
@@ -67,6 +80,12 @@ void endToEnd() {
         metric ->
             assertTypedGauge(
                 metric, "jvm.memory.pool.used", "current memory pool usage", "by", gcLabels),
-        metric -> assertGauge(metric, "jvm.threads.count", "number of threads", "1"));
+        metric ->
+            assertSum(
+                metric,
+                "process.runtime.jvm.threads.count",
+                "number of threads",
```

Comment:
I'm probably missing something, do u know why this isn't:

```suggestion
                "Number of executing threads",
```